sdk/trace/batch_span_processor_test.go

```diff
@@ -256,4 +255,9 @@ func TestBatchSpanProcessorShutdown(t *testing.T) {
 		t.Error("Error shutting the BatchSpanProcessor down\n")
 	}
 	assert.Equal(t, 1, bp.shutdownCount)
+
+	bspCopy, ok := bsp.(*batchSpanProcessor)
+	if ok {
+		assert.Nil(t, bspCopy.e)
+	}
```

Comment:
```suggestion
	assert.True(t, ok, "bsp is a *batchSpanProcessor")
	assert.Nil(t, bspCopy.e, "exporter is now nil")
```

To be safe here it's probably best to assert that `ok` is `true` rather than skipping the `nil` assertion if it isn't.